public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -194,6 +194,7 @@ export const ServiceAccountsListPageUnconnected = ({
             options={[
               { label: 'All', value: ServiceAccountStateFilter.All },
               { label: 'With expired tokens', value: ServiceAccountStateFilter.WithExpiredTokens },
+              { label: 'Managed', value: ServiceAccountStateFilter.Managed },
```

Comment:
Thought: I wonder if it would make sense to filter managed service accounts out of the default service account view? As I imagine users won't interact with them much, so it makes sense to hide them by default. No strong feelings on it though.

Reply:
I like the idea. Maybe we can put a pin on it and tackle it in another PR? (As I feel it'd add another filter)

Another thing to consider, we are probably talking about 10 external service accounts top (if you have 10 plugins installed that need one :D)

Reply:
I like this idea. I agree with @gamab, let's tackle this in another PR.

My thinking is, we need to be more transparent with the users. I don't think the filters tell us there are any external SA until we click on it. I think it'd be really cool for the filters to append the total count of the SA queried.